Examples/Framework/include/ActsExamples/Framework/Sequencer.hpp

```diff
@@ -58,7 +58,8 @@ class SequenceConfigurationException : public std::runtime_error {
 class Sequencer {
  public:
   struct FpeMask {
-    std::string loc;
+    std::string file;
+    std::pair<unsigned int, unsigned int> lines;
```

Comment:
```suggestion
    std::pair<std::size_t, std::size_t> lines;
```
nitpick: or `std::uint32_t`

Reply:
no particular - I guess mostly to ensure the size and consistency e.g. we use `std::size_t count;` below

but its only a small preference from my side. not blocking in any way

Reply:
I'll change it.